Cargo.toml

```diff
@@ -16,7 +16,11 @@ dynamic_link = []
 pdfium_build = []
 
 [dependencies]
-link-cplusplus = {version = "1.0", optional=true}
+link-cplusplus = { version = "1.0", optional = true }
 
 [build-dependencies]
-bindgen = {version = "0.54", optional=true}
+bindgen = { version = "0.54", optional = true }
```

Comment:
Why is it that bindgen is optional btw? The library seems useless without it.

Reply:
It is because the user may not always want or need to generate new bindings. This should only be needed if the existing bindings are out of date or for other reasons not working for the user. Since it requires the user to have Clang installed and also brings in additional dependencies, my hope to avoid it in most situations.

Reply:
>It is because the user may not always want or need to generate new bindings.

I'm not too familiar with the intricacies of bindgen, but doesn't the bindings depend on the target? I.e. you can't persist them in this repository and use them across targets. For example the size of the various C integer types change based on the target. Or am I misunderstanding something?